pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -132,18 +137,6 @@ func (r *Reconciler) reconcile(ctx context.Context, cronjob *v1alpha1.CronJobSou
 
 	cronjob.Status.InitializeConditions()
 
-	_, err := cron.ParseStandard(cronjob.Spec.Schedule)
-	if err != nil {
-		cronjob.Status.MarkInvalidSchedule("Invalid", "")
-		return fmt.Errorf("invalid schedule: %v", err)
-	}
-	cronjob.Status.MarkSchedule()
-
-	if cronjob.Spec.Sink == nil {
-		cronjob.Status.MarkNoSink("Missing", "Sink missing from spec")
-		return errors.New("spec.sink missing")
-	}
-
 	sinkObjRef := cronjob.Spec.Sink
```

Comment:
Don't we still need to check if `sinkObjRef` is nil?

Reply:
this is done by the admission controller. Maybe we should also change the Sink type as I believe this is a mandatory field now.